addons/website_hr_recruitment/migrations/17.0.1.1/pre-migrate.py

```diff
@@ -0,0 +1,12 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+def migrate(cr, version):
+
+    # Remove the csrf_token from the website_hr_recruitment.apply view
+    cr.execute(r"""
+        UPDATE ir_ui_view
+        SET arch_db = REGEXP_REPLACE(arch_db::text, '<input id=\\"csrf_token\\" type=\\"hidden\\" name=\\"csrf_token\\" t-att-value=\\"request.csrf_token\(\)\\"/>', '', 'g')::jsonb
```

Comment:
Can't you match the column around the input too? Maybe also with something less restrictive, which handles potential attribute change on both elements:
```javascript
/<div[^>]+class="[\w-]*col-sm[\w-]*"[^>]*>\s*<input[^/]+id="csrf_token"[^/]+/>\s*</div>/
```
(did that without testing and using regex as I would in JS, not sure it's the same here).

Reply:
Sure I can do that I adapted the querry to make it less restrictive with this RegEx
`'<div[^>]*?class=\\"col-sm\\"[^>]*?>[^<]*?<input[^>]*?id=\\"csrf_token\\"[^>]*?/>[^<]*?</div>'`

Reply:
- Why the use of `*?` instead of `*` or `+` ?
- Why not handling the class addition on the col like my regex suggested?

Reply:
- In this case + and * would match the same way in this case because normally odoo would raise an error if a div was malformed in that manner. But for the *? effectively I could just used * since greedy or not greedy matching shouldn't change anything in that case because I prohibited the ">" or the "<"
- Now that I think a bit more about it I don't think the class matching is necessary at all since this div should always be the first one that is before the input 

But I'll readapt it and test it today to be more correct

Reply:
@qsm-odoo ?